sdk/confidentialledger/azure-confidentialledger/README.md

```diff
@@ -80,175 +114,181 @@ Users are managed directly with the Confidential Ledger instead of through Azure
+Azure Confidential Ledger is built on Microsoft Research's open-source [Confidential Consortium Framework (CCF)][ccf]. Under CCF, applications are managed by a consortium of members with the ability to submit proposals to modify and govern application operation. In Azure Confidential Ledger, Microsoft Azure owns a an operator member identity that allows it to perform governance and maintenance actions like replacing unhealthy nodes in the Confidential Ledger and upgrading the enclave code.
 
 ## Examples
 This section contains code snippets covering common tasks:
-* [Append a ledger entry](#append-entry "Append a ledger entry")
-* [Get a receipt](#get-receipt "Get a receipt")
-* [Using sub-ledgers](#using-sub-ledgers "Using sub-ledgers")
-* [Retrieving ledger entries](#retrieving-ledger-entries "Retrieving ledger entries")
-* [Making a ranged query](#making-a-ranged-query "Making a ranged query")
-* [Managing users](#managing-users "Managing users")
-* [Using certificate authentication](#using-certificate-authentication "Using certificate authentication")
-* [Verifying service details](#verifying-service-details "Verifying service details")
-* [Asynchronously get a ledger entry](#asynchronously-get-a-ledger-entry "Asynchronously get a ledger entry")
-* [Asynchronously get a range of ledger entries](#asynchronously-get-a-range-of-ledger-entries "Asynchronously get a range of ledger entries")
-
 
 ### Append entry
 Data that needs to be stored immutably in a tamper-proof manner can be saved to Azure Confidential Ledger by appending an entry to the ledger.
 
 ```python
-first_append_result = ledger_client.append_to_ledger(entry_contents="Hello world!")
-print(first_append_result.transaction_id)
+post_entry_result = ledger_client.post_ledger_entry(
+        {"contents": "Hello world!"}
+    )
+transaction_id = post_entry_result["transactionId"]
+print(transaction_id)
 ```
 
 Since Confidential Ledger is a distributed system, rare transient failures may cause writes to be lost. For entries that must be preserved, it is advisable to verify that the write became durable. Waits are blocking calls.
+
 ```python
-from azure.confidentialledger import TransactionState
-ledger_client.wait_until_durable(transaction_id=first_append_result.transaction_id)
-assert ledger_client.get_transaction_status(
-    transaction_id=first_append_result.transaction_id
-).state is TransactionState.COMMITTED
-
-# Alternatively, a client may wait when appending.
-append_result = ledger_client.append_to_ledger(
-    entry_contents="Hello world, again!", wait_for_commit=True
+wait_poller = ledger_client.begin_wait_for_commit(transaction_id)
```

Comment:
all samples should be fully runnable, so you still need to include the imports and client initialization